@navikt/core/react/src/form/combobox/ClearButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { XMarkIcon } from "@navikt/aksel-icons";
+
+interface ClearButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  handleClear: (event: any) => void;
+  clearButtonLabel?: string;
+}
+
+export const ClearButton: React.FC<ClearButtonProps> = ({
+  handleClear,
+  clearButtonLabel,
+}) => {
+  return (
+    <button
+      type="button"
+      onClick={handleClear}
+      className="navds-combobox__button-clear"
+    >
+      <span className="navds-sr-only">
+        {clearButtonLabel ? clearButtonLabel : "Tøm"}
+      </span>
+      <XMarkIcon aria-hidden width="20" height="20" />
+    </button>
+  );
+};
+
+export default ClearButton;
```

Comment:
Out-of-scope for denne PR-en sikkert, men kan "ClearButton" brukt i `<Search />` også trekkes ut slik at begge bruker samme komponent. Tok bare en raskt titt, men tror API-et er relativt likt mellom dem (ikke sikker på designet)

Reply:
Enig, men i en egen PR seinere? 😅

Reply:
Ja kan være en TODO for fremtiden haha 👍